formatLinkHeaderRelations.mjs

```diff
@@ -1,5 +1,12 @@
+// @ts-check
+
 import formatLinkHeaderRelation from "./formatLinkHeaderRelation.mjs";
 
+/**
+ * Formats an array of resources as a Link header value.
+ * @param {Array<string>} resources The resources to format.
```

Comment:
What is a "resource"? What is in the strings?

```suggestion
 * @param {Array<string>} resources Resources to format.
```

For all JSDoc descriptions, it's generally best to avoid leading with the word `The `, because it doesn't add any value.